addons/website_forum/static/src/scss/website_forum.scss

```diff
@@ -243,4 +263,36 @@ img.o_forum_avatar_big {
     #o_wforum_forums_index_list & {
         min-height: 100px;
     }
+
+    & img {
+        height: 100%;
+        width: 100%;
+        object-fit: cover;
+    }
+}
+
+#o_wforum_forums_index_list {
+
+    @media (max-width: 768px) {
+        .o_forum_row {
+            flex-direction: column !important;
+        }
+
+        .o_forum_image_container {
+            height: 150px;
+        }
+    }
 }
+
+.o_forum_list_view {
+    height: 150px;
+}
+
+.opacity-0 img {
+    opacity: 0;
+}
+
+.accordion > .card:first-of-type:last-of-type {
+    border-bottom: 1px solid rgba(0, 0, 0, 0.125);
+    border-radius: 0.25rem;
+}
```

Comment:
Missing new line at the end of this file, your editor should add those automatically.